package.json

```diff
@@ -163,11 +163,7 @@
     "codegen": "build/run-node build/generate-style-code.js && build/run-node build/generate-struct-arrays.js"
   },
   "files": [
-    "build/",
```

Comment:
It's a bit of a risk for us to change the content of the package as users may have a hard dependency on these.

>build scripts are useless without dev dependencies

Agreed, though it's really hard to know what type of workflow some library users have beforehand, maybe the dev dependencies are added by them. 

>all files in dist are published

I think this is one would be a no-op and we preferred being explicit, changing this shouldn't have any impact and prevents accidental addition.

>.flowconfig and flow-typed in packages are not used by flow

I can't confirm this one. Do you know @arindam1993 ?

Reply:
yes this is correct